### ISSUE TITLE
Change input types for kube_get and kube_capture

### ISSUE THREAD

**Issue by srm09** [vmware-tanzu/crash-diagnostics#119]
Currently the directives `kube_get()` and `kube_capture()` both accept a bunch of different flags (`labels`/`namespaces`/`groups`/`kinds`/`names`/`versions`/`labels`), each of which can be a string value or a list.

Update the implementation of both to accept each of the inputs as a list type only.

Example:
```python
kube_get(labels="abcd", namespaces="default kube-system")
kube_capture(kinds="deployments", namespaces="default kube-system")
``` 

The above diagnostic file snippet should throw an error
